packages/field/src/FocusMixin.js

```diff
@@ -31,17 +31,17 @@ export const FocusMixin = dedupeMixin(
        *   return this.inputElement === document.activeElement;
        */
       get focused() {
-        return this.classList.contains('state-focused');
```

Comment:
can we maybe use a property reflected in an attribute instead of writing this boilerplate code, like we have for `touched` and `dirty`?

also in the new lion-calendar we are using this now with just `[%name%]` instead of `[state-%name%]`

in terms of compatibility it will be a similar change since you are changing the API regardless (from classes to attributes)

Reply:
@daKmoR or shall we also use `[state-%name%]` in the lion-calendar?

Reply:
I would say it should just be `focused` as property / attribute then this getter goes naturally away :hugs:

Reply:
let's please go for `[focused]`